go/vm/lfvm/converter.go

```diff
@@ -528,7 +528,7 @@ func createOpToOpMap() []OpCode {
 	res[evm.COINBASE] = COINBASE
 	res[evm.TIMESTAMP] = TIMESTAMP
 	res[evm.NUMBER] = NUMBER
-	res[evm.DIFFICULTY] = DIFFICULTY
+	res[evm.DIFFICULTY] = PREVRANDAO //TODO: update with new geth version
```

Comment:
is todo still valid?

Reply:
Yes, once the geth version is updated this line changes to `res[evm.PREVRANDAO] = PREVRANDAO`.